src/admin/class-event-dates.php

```diff
@@ -28,6 +28,12 @@ public function __construct() {
 		if ( ! wp_next_scheduled( 'openagenda_cron_event_weekly' ) ) {
 			wp_schedule_event( strtotime( 'NEXT SUNDAY' ), 'weekly', 'openagenda_cron_event_weekly' );
 		}
+
+		add_action( 'openagenda_update_next_event_date', array ( $this, 'update_next_event_date' ));
```

Comment:
Algemeen: Wij hanteren de WordPress Coding Standards, ik zie in deze class wat kleine dingen die niet helemaal kloppen. Als je in de plugin directory `composer install` runt, kun je daarna de code laten checken met het commando:
`./vendor/bin/phpcs --standard=phpcs.xml.dist .`
Volgens mij kunnen de meeste fouten automatisch voor je worden gefixt met het commando:
`./vendor/bin/phpcbf --standard=phpcs.xml.dist .`